### DISCUSSION TITLE
Jellyseer stuck on pending approval?

### DISCUSSION THREAD

**Discussion by Brooklyn727** [seerr-team/seerr#514] [General]
As the title states, I am the admin and the only person who uses this on my home network. I will request a new show, and it normally would tell me that it is automatically approved, but now, it just sends the request, and I am unable to approve it. I have restarted the container and logged in and out of Jellyseer to no avail. Is there any way to troubleshoot this?

Pic for reference: https://imgur.com/ykevWUz

**Comment by fallenbagel** [comment]
Sounds like you lost admin privileges?

Check if you have access to settings page

**Reply by happy1234567** [reply]
I cannot find any support for JellySeer.   I have 4 OMV servers.  .everyone of them suddenly  does the exact same thing... I cannot approve.... I cannot administer the JellySeer.  Please help. I only made the one admin login, what's going on? .  I have full admin rights on all of them. I removed and reinstall the docker container...same issue on all 4 servers

**Reply by fallenbagel** [reply]
Can you hop on discord? Removing and reinstalling docker container will STILL use the old config folder. pretty sure your userid changed and you lost admin privileges from your account on jellyseerr.

we do our support on discord. The link is there on readme.
